dev-util/SPIRV-Headers/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>x11@gentoo.org</email>
+		<name>X11</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>commendsarnex@gmail.com</email>
```

Comment:
If you want to proxy-maintain it, you need proxy-maint team below you.

Reply:
i will work with the x11 team to review for now